test/spec/modules/openxAnalyticsAdapter_spec.js

```diff
@@ -1,443 +1,630 @@
+        sinon.stub(utils, 'getWindowLocation').returns({search: '?' +
+            'utm_campaign=test-campaign-name&' +
+            'utm_source=test-source&' +
+            'utm_medium=test-medium&' +
+            'utm_content=test-content&' +
+            'utm_term=test-term'
+        });
 
-        const queryData = getQueryData(server.requests[0].url);
-        expect(queryData).to.include({
-          'hb.cur': 'bitcoin'
+        openxAdapter.enableAnalytics({
+          options: {
+            ...DEFAULT_V2_ANALYTICS_CONFIG,
+            campaign: {
+              name: 'test-config-name',
+              source: 'test-config-source',
+              medium: 'test-config-medium'
+            }
+          }
         });
-      });
 
-      it('should not be in the query string if currency is not set', function() {
         simulateAuction([
           [AUCTION_INIT, auctionInit],
-          [BID_REQUESTED, bidRequestedOpenX],
-          [BID_RESPONSE, bidResponseOpenX]
+          [SLOT_LOADED],
         ]);
-
-        const queryData = getQueryData(server.requests[0].url);
-        expect(queryData).to.not.have.key('hb.cur');
+        clock.tick(SLOT_LOAD_WAIT_TIME);
+        auction = JSON.parse(server.requests[0].requestBody)[0];
+
+        expect(auction.campaign.name).to.equal('test-config-name');
+        expect(auction.campaign.source).to.equal('test-config-source');
+        expect(auction.campaign.medium).to.equal('test-config-medium');
+        expect(auction.campaign.content).to.equal('test-content');
+        expect(auction.campaign.term).to.equal('test-term');
       });
     });
 
-    describe('hb.dcl, hb.dl, hb.tta, hb.ttr', function() {
-      it('should be in the query string if browser supports performance API', function() {
-        const timing = {
-          fetchStart: 1540944528000,
-          domContentLoadedEventEnd: 1540944528010,
-          loadEventEnd: 1540944528110
-        };
-        const originalPerf = window.top.performance;
-        window.top.performance = { timing };
+    describe('when there are bid requests', function () {
+      let auction;
+      let openxBidder;
+      let closexBidder;
 
-        const renderTime = 1540944528100;
-        sinon.stub(Date, 'now').returns(renderTime);
+      beforeEach(function () {
+        openxAdapter.enableAnalytics({options: DEFAULT_V2_ANALYTICS_CONFIG});
 
         simulateAuction([
           [AUCTION_INIT, auctionInit],
+          [BID_REQUESTED, bidRequestedCloseX],
           [BID_REQUESTED, bidRequestedOpenX],
-          [BID_RESPONSE, bidResponseOpenX]
+          [SLOT_LOADED],
         ]);
+        clock.tick(SLOT_LOAD_WAIT_TIME * 2);
+        auction = JSON.parse(server.requests[0].requestBody)[0];
+        openxBidder = auction.adUnits[0].bidRequests.find(bidderRequest => bidderRequest.bidder === 'openx');
+        closexBidder = auction.adUnits[0].bidRequests.find(bidderRequest => bidderRequest.bidder === 'closex');
```

Comment:
See example here.

Reply:
Thanks for bringing this up.  I wasn't aware that find was something I'd need to use the polyfill on in tests.  I updated the tests accordingly.